tools/src/commands/GenerateStoriesApp.ts

```diff
@@ -0,0 +1,97 @@
+import { Command } from '@expo/commander';
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs-extra';
+import path from 'path';
+
+import { EXPO_DIR, PACKAGES_DIR } from '../Constants';
+import {
+  GenerateBareAppOptions,
+  action as generateBareApp,
+  getDirectories,
+  getPackagesToSymlink,
+  symlinkPackages,
+} from './GenerateBareApp';
+
+async function action(packageNames: string[], options: GenerateBareAppOptions) {
+  const { projectDir, workspaceDir } = getDirectories(options);
+  await generateBareApp(packageNames, options);
+  await addNativeDependencies({ projectDir });
+  await createMetroConfig({ projectRoot: projectDir, ...options });
+  await rewriteAppFile({ projectDir, appName: options.name });
+
+  const packagesToSymlink = await getPackagesToSymlink({ packageNames, workspaceDir });
+  await symlinkPackages({ packagesToSymlink, projectDir });
+}
+
+async function addNativeDependencies({ projectDir }: { projectDir: string }) {
+  const pkgPath = path.resolve(projectDir, 'package.json');
+  const pkg = await fs.readJSON(pkgPath);
+
+  const bundledNativeModules = require(path.resolve(
+    PACKAGES_DIR,
+    'expo',
+    'bundledNativeModules.json'
+  ));
+
+  pkg.dependencies['react-native-safe-area-context'] =
+    bundledNativeModules['react-native-safe-area-context'];
+  pkg.dependencies['react-native-screens'] = bundledNativeModules['react-native-screens'];
+  pkg.dependencies['react-native-svg'] = bundledNativeModules['react-native-svg'];
+  pkg.dependencies['react-native-gesture-handler'] =
+    bundledNativeModules['react-native-gesture-handler'];
```

Comment:
Whenever I have to write the same thing 3rd time I create a helper mechanism doing that.
```suggestion
  [
    'react-native-safe-area-context',
    'react-native-screens',
    'react-native-svg',
    'react-native-gesture-handler'
  ].forEach(packageName => {
    pkg.dependencies[packageName] = bundledNativeModules[packageName];
  });
```